pkgs/test/nixpkgs-check-by-name/src/nixpkgs_problem.rs

```diff
@@ -55,7 +90,7 @@ pub enum NixpkgsProblem {
     UnresolvableSymlink {
         relative_package_dir: PathBuf,
         subpath: PathBuf,
-        io_error: io::Error,
+        io_error: String,
```

Comment:
I changed this because I had to make `NixpkgsProblem` a `Clone`, because of this new use:

https://github.com/NixOS/nixpkgs/blob/64da6178bf10b92f57352d7d0cfca7eebbd527df/pkgs/test/nixpkgs-check-by-name/src/ratchet.rs#L129-L139

And I can't find a good way to avoid that just yet